mod-source-record-manager-server/src/main/java/org/folio/rest/impl/InitAPIImpl.java

```diff
@@ -46,7 +46,7 @@ public class InitAPIImpl implements InitAPI {
   @Value("${srm.kafka.StoredMarcChunkConsumer.instancesNumber:3}")
   private int storedMarcChunkConsumerInstancesNumber;
 
-  @Value("${srm.kafka.DataImportConsumersVerticle.instancesNumber:3}")
+  @Value("${srm.kafka.DataImportConsumersVerticle.instancesNumber:1}")
```

Comment:
@SerhiiNosko does this change will not affect performance on production envs ?

Reply:
It should not affect performance in the negative side, because each consumer should process single partition. It does not make sense to increase number of consumers if we have the same number of partitions.
For scaling if we add more topic partitions - need to add more instances.

Reply:
Got it!Thank you.